tests/Doctrine/Tests/ORM/Proxy/ProxyFactoryTest.php

```diff
@@ -75,6 +75,26 @@ public function testReferenceProxyDelegatesLoadingToThePersister()
         $proxy->getDescription();
     }
 
+    public function testSkipMappedSuperClassesOnGeneration()
```

Comment:
> Please add @group 6625 on the docblock

Although this test case is unrelated with #6625 actually? I just found that test case is missing, thus I added. :)